qa/workunits/fs/misc/layout_vxattrs.sh

```diff
@@ -105,6 +105,21 @@ getfattr -n ceph.file.layout.stripe_count dir/file | grep -q 8
 getfattr -n ceph.file.layout.object_size dir/file | grep -q 10485760
 getfattr -n ceph.file.layout.pool_namespace dir/file | grep -q dirns
 
+setfattr -x ceph.dir.layout.pool dir 2>&1 | grep "setfattr: dir: Invalid argument"
+setfattr -x ceph.dir.layout.pool_id dir 2>&1 | grep "setfattr: dir: Invalid argument"
+setfattr -x ceph.dir.layout.pool_name dir 2>&1 | grep "setfattr: dir: Invalid argument"
+setfattr -x ceph.dir.layout.stripe_unit dir 2>&1 | grep "setfattr: dir: Invalid argument"
+setfattr -x ceph.dir.layout.stripe_count dir 2>&1 | grep "setfattr: dir: Invalid argument"
+setfattr -x ceph.dir.layout.object_size dir 2>&1 | grep "setfattr: dir: Invalid argument"
+
+setfattr -x ceph.file.layout.pool dir/file 2>&1 | grep "setfattr: dir/file: Invalid argument"
+setfattr -x ceph.file.layout.pool_id dir/file 2>&1 | grep "setfattr: dir/file: Invalid argument"
+setfattr -x ceph.file.layout.pool_name dir/file 2>&1 | grep "setfattr: dir/file: Invalid argument"
+setfattr -x ceph.file.layout.stripe_unit dir/file 2>&1 | grep "setfattr: dir/file: Invalid argument"
+setfattr -x ceph.file.layout.stripe_count dir/file 2>&1 | grep "setfattr: dir/file: Invalid argument"
+setfattr -x ceph.file.layout.object_size dir/file 2>&1 | grep "setfattr: dir/file: Invalid argument"
+setfattr -x ceph.file.layout.pool_namespace dir/file
+
```

Comment:
Shouldn't we set these vxattrs using `setfattr -n` first then try remove them?

Reply:
and maybe add getfattr lines to check them (commented for now since the issue with test machines) ?

Reply:
> Shouldn't we set these vxattrs using `setfattr -n` first then try remove them?
I think that case is already handled up higher in the test. xattr is set, then read back. The values here should be set values or inherited values (dir or file cases).

My intention for dir/file is to try to remove xattrs in cases where it doesn't make sense. So for instance, you shouldn't be able to remove layout information from a single layout attribute except for pool_namespace. In the docs, it appears the intended way is to remove all layout information from a dir or file at once[1]. 

1- https://docs.ceph.com/en/quincy/cephfs/file-layouts/#clearing-layouts

Reply:
This is some insightful info; adding a comment about `pool_namespace` might be a good thing to do IMO.